src/config.c

```diff
@@ -33,6 +33,8 @@
 /*----------------------------------------------------------------------------*/
 /*                            File Scoped Variables                           */
 /*----------------------------------------------------------------------------*/
+pthread_mutex_t config_mut=PTHREAD_MUTEX_INITIALIZER;
```

Comment:
is this under wan fail over flag or generic also mutex required?

Reply:
okk